synapse/federation/transport/server/_base.py

```diff
@@ -180,7 +180,10 @@ def _parse_auth_header(header_bytes: bytes) -> Tuple[str, str, str, Optional[str
     """
     try:
         header_str = header_bytes.decode("utf-8")
-        params = re.split(" +", header_str)[1].split(",")
+        params = re.split(
+            r"\s*,\s*",
```

Comment:
`\s` typically matches the following characters:

- Space (SP)
- Tab (HT)
- Carriage return (CR)
- Line feed (LF)
- Form feed (FF)
- Vertical tab (VT)

which is more than the SP and HT defined by RFC7235. Instead, we can write:

```suggestion
        space_or_tab = "[ \t]"
        params = re.split(
            rf"{space_or_tab}*,{space_or_tab}*",
```

Reply:
Yes was not sure if it was preferred to be lax or not. 
Will change it :).